### ISSUE TITLE
Rename runtime import namespace to `locutus-runtime`

### ISSUE THREAD

**Issue by brancz** [brancz/locutus#19]
Currently locutus uses the `generic-operator` namespace, which is an artifact of the proof-of-concept stage of this project. I think `locutus-runtime` would be more representative of today's state.

**Comment by s-urbaniak** [comment]
agreed! @paulfantom, maybe a something for you as it affects the jsonnet part?

**Comment by brancz** [comment]
FWIW for the implementation of #18 I already went with this new proposed prefix/namespace. So really all we should do is branch off a version of locutus now, and then do the change here: https://github.com/brancz/locutus/blob/master/render/jsonnet/jsonnet.go#L55
